### PR TITLE
DM-31881: Implement Processing of corner wavefront sensor data in MTAOS.

### PR DESCRIPTION
**Pull request by tribeiro**

### CODE REVIEW

File: tests/wep_integration/test_lsstcam_cwfs.py [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+from lsst.ts.ofc import OFCData
+
+from lsst.daf import butler as dafButler
+
+from lsst.ts.wep.Utility import writeCleanUpRepoCmd, runProgram
+from lsst.ts.wep.Utility import getModulePath as getModulePathWep
+
+
+@pytest.mark.integtest
+class TestComCam(unittest.IsolatedAsyncioTestCase):
```
Comment (teweitsai) [inline tests/wep_integration/test_lsstcam_cwfs.py:39]: I think the class name should not be `TestComCam`.

**Review by teweitsai — APPROVED** [review]
Looks good. Please help to improve the document. Please help to explain the separation of unit tests in Jenkinsfile is it is necessary in `versionHistory.rst`. Thanks!

File: doc/versionHistory.rst [hunk 2]
```diff
@@ -2,6 +2,23 @@
 Version History
 ===============
 
+v0.10.0
+-------
+
+* In Jenkinsfile, separate running tests marked as integtest and csc_integtest from the other unit tests. 
+  Run non-marked tests first and, if successful, run integtest and csc_integtest respectively.
```
[the remark this answers is not in the document]
Comment (tribeiro) [reply doc/versionHistory.rst:9]: There are a couple reasons to run them separately. One of them is that these tests take quite a while to execute. If the unit tests fails, I am not sure I want to wait ~10-20 minutes for the job to complete and occupy resources.

[the remark this answers is not in the document]
Comment (teweitsai) [reply doc/versionHistory.rst:9]: This makes sense. Please help to explain this in the `versionHistroy.rst` and update the `README.md` to remind the future's developer. Thanks!

**Pull request merged**